### DISCUSSION TITLE
Welcome to WFN Discussions!

### DISCUSSION THREAD

**Discussion by wokhan** [wokhansoft/WFN#119] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wokhan** [comment]
And yeah, out of laziness I kept the default template for this first discussion 🙄 😄

**Reply by GregZero** [reply]
Hello, I'm Greg, an IT enthusiast and average user.
Thank you for this tool that I have been using since the first versions and which is very promising and could be very useful.
Well done !
